static/app/components/tooltip.tsx

```diff
@@ -479,4 +388,30 @@ const TooltipArrow = styled('span')`
   }
 `;
 
+interface AcceptanceTestTooltipProxyProps extends TooltipProps {
```

Comment:
Shouldn't we actually be exporting these props? since `disableForVisualTest` is part of the external tooltip props type?

Reply:
No, right now no component actually requires importing the props, so we can keep em unexported. They are still part of the API, just not exported till something needs it

Reply:
But isn't this "incorrect"? Why not rename `TooltipProps` above to something else?

Isn't this already confusing since this component isn't even named `AcceptanceTestTooltipProxy` anymore?

Reply:
Oh yeah, I thought I kept it as AcceptanceTestTooltipProxy, dumb from me. Will change

Reply:
+1 to keeping the props title consistent with its component.
would prefer to rename the above `TooltipProps` to something else, even if it's a weird title like DoNotUseTooltipProps or whatnot.

I don't want to think about which props correlate to which components.